sourced/ml/cmd_entries/repos2roles_and_ids.py

```diff
@@ -14,7 +14,8 @@ def repos2roles_and_ids_entry(args):
 
     Ignition(engine, explain=args.explain) \
         .link(HeadFiles()) \
-        .link(UastExtractor(languages=args.languages)) \
+        .link(LanguageSelector(languages=args.languages)) \
```

Comment:
you should fix it for Apollo also. For example https://github.com/src-d/apollo/blob/master/apollo/bags.py#L82

Reply:
thks, I forgot that part, adding it to my PR in apollo ^^